src/quo2/core.cljs

```diff
@@ -188,6 +189,9 @@
 (def search-input quo2.components.inputs.search-input.view/search-input)
 (def title-input quo2.components.inputs.title-input.view/title-input)
 
+;;;; Keyboard
```

Comment:
nit: this should be upper case for consistency and also matching the area name, i.e
`;;;; NUMBERED KEYPOARD`

Reply:
Done

Reply:
Just jumping in to share my perspective.

I wish we didn't follow this convention. It's also not in the Clojure Style Guide https://guide.clojure.style/#four-semicolons-for-heading-comments

For an Emacs user, heading comments can be easily syntax highlighted (even by level). Vim should probably have this capability but I'm not using it anymore.

But I totally get why devs using other editors opted for using the *screaming case* to differentiate between normal comments and heading comments :)

Reply:
tbh I don't think there's much of a convention there, we can just change that file to use 4 afaik 👍

Reply:
@J-Son89, I meant the case of the text is all uppercased. I know this is not important, who cares about the case of the heading right? But anyway it's not in the Clojure style guide.

Reply:
Ah apologies, yeah I think we can just update this so?

Reply:
I would say so, our guidelines say we follow the CSG unless stated otherwise.